liota/dcc/vrops.py

```diff
@@ -39,11 +39,12 @@
 import sys
 import threading
 from time import timezone
+import ConfigParser
 
 from liota.dcc.dcc_base import DataCenterComponent
 from helix_protocol import HelixProtocol
 from liota.core.metric_handler import Metric
-from liota.utilities.utility import getUTCmillis
+from liota.utilities.utility import getUTCmillis, findLiotaConfigFullPath
```

Comment:
It is not apparent that findLiotaConfigFullPath is a class. Generally, class names should be UpperCaseCamelCase. Also, class names are generally nouns and method names are actions like 'find'. Can you please change that?